src/App.jsx

```diff
@@ -14,45 +17,91 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const SORT_BY_LENGTH = 'length';
+const SORT_BY_ALPHABET = 'alphabetically';
+
+function getPreparedGoods(goods, sorted, reversed) {
```

Comment:
```suggestion
function getPreparedGoods(
  goods, 
  sortType, 
  isReversed,
) {
```
It would be great to provide more descriptive and meaningful names